show/main.py

```diff
@@ -595,6 +659,52 @@ def uptime():
     """Show system uptime"""
     run_command('uptime -p')
 
+@cli.command()
+def clock():
+    """Show date and time"""
+    run_command('date')
+
+@cli.command('system-memory')
+def system_memory():
+    """Show memory information"""
+    command="free -m"
+    run_command(command)
+
+@click.group(cls=AliasedGroup, default_if_no_args=False)
```

Comment:
Change to `@cli.group(cls=AliasedGroup, default_if_no_args=False)` and remove `cli.add_command(vlan)` on line 678.

Reply:
done